### ISSUE TITLE
Error: Could not find or load main class org.stjude.compbio.sam.TweakSam

### ISSUE THREAD

**Issue by lentaing** [stjude/XenoCP#11]
Hi,

I'm getting a class not found error when trying to run xenoCP.  My current CLASSPATH is set to:
CLASSPATH=[xenoCP install directory]/build/install/xenocp/lib:.

Can you help me locate this class?  Is my CLASSPATH set correctly?

Regards,
Len Taing

**Comment by lentaing** [comment]
UPDATE: I found that java.sh was not properly handling the CLASSPATH environment variable.  I had to fix it by hard-coding my CLASSPATH into java.sh before the case statements.

**Comment by adthrasher** [comment]
I believe the correct CLASSPATH setting would be: 
CLASSPATH=[xenoCP install directory]/build/install/xenocp/lib/*:.

The directory contains JAR files, so the wildcard character is necessary for Java to properly search.

**Comment by lentaing** [comment]
Hi Andrew,

Thanks for your reply.  I've tried the path with the wildcard--but it did not work.  My work around, posted above, seemed to have resolved the issue.

I'm not sure why CLASSPATH is not being properly set in java.sh--my best guess is that it could be related to the conda environment I created for the xenoCP dependencies or perhaps is related to how cromwell interacts with conda...I'm not sure.

Regards,
Len
